### ISSUE TITLE
[RadDataForm] iOS caps displayed default number values at 100

### ISSUE THREAD

**Issue by vcooley** [telerik/nativescript-ui-feedback#177]
After upgrading my nativescript-ui-pro package, I'm getting incorrect values displayed for numbers above 100. All the values display as 100, even though the source is set to have them higher. For instance:
![image](https://cloud.githubusercontent.com/assets/12551141/25754421/b4d78252-3184-11e7-981d-546c30cf09e3.png)
when logging the source I get: `"zipCode":75080`

So it appears to not change the values themselves, but only the displayed values.

**Comment by tgpetrov** [comment]
Hi @vcooley ,

Can you share with us a runnable project or code snippets of how the DataForm is setup. Is this issue for all editors or only with specific properties?

**Comment by vcooley** [comment]
Template
	<RadDataForm row="0" [source]="source" [metadata]="metadata" ">
		<TkEntityProperty tkDataFormProperty index="0" name="firstName" displayName="First Name"></TkEntityProperty>
		<TkEntityProperty tkDataFormProperty index="1" name="lastName" displayName="Last Name"></TkEntityProperty>
		<TkEntityProperty tkDataFormProperty index="2" name="amount" hintText="Amount Due" displayName="Amount Due"></TkEntityProperty>
	</RadDataForm>


Component
	export class dataFormComp implements OnInit {
		source: any;
		ngOnInit() {
			source = {
				firstName: Ter,
				lastName: Stegen,
				amount: 245,
			}
		}
	}

Metadata
	export const paymentChoiceMetadata = {
		isReadOnly: false,
		commitMode: 'immediate',
		validationMode: 'immediate',
		propertyAnnotations: [
			{
				name: 'firstName',
				index: 0,
				displayName: 'First Name',
				hintText: 'First Name',
				editor: 'Text'
			},
			{
				name: 'lastName',
				index: 1,
				displayName: 'Last Name',
				hintText: 'Last Name',
				editor: 'Text'
			},
			{
				name: 'amount',
				index: 2,
				displayName: 'Amount Due',
				hintText: 'Amount Due',
				editor: 'Number'
			},
		]
	};


This currently causes my amount field to display 100, even though the source value is set to a higher number.

Thanks!

**Comment by tgpetrov** [comment]
Hi @vcooley,

Thanks for the snippet, I confirm the issue which occurs when using the metadata feature.
As a workaround you can declare the editor type in the html:
```
<TKEntityProperty tkDataFormProperty index="2" name="amount" hintText="Amount Due" displayName="Amount Due">
	<TKPropertyEditor tkEntityPropertyEditor type="Number"></TKPropertyEditor>
</TKEntityProperty>
```

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-dataform@3.8.
